src/v2/Apps/FairOrginizer/fairOrganizerRoutes.tsx

```diff
@@ -17,5 +18,17 @@ export const fairOrganizerRoutes: AppRouteConfig[] = [
     prepare: () => {
       FairOrganizerApp.preload()
     },
+    prepareVariables: params => {
+      return {
+        id: params.slug,
+      }
+    },
+    query: graphql`
+      query fairOrganizerRoutes_FairQuery($id: String!) {
+        fair(id: $id) {
+          ...FairOrganizerApp_fair
+        }
+      }
+    `,
```

Comment:
Note that if these `$id`s were `$slug` then we wouldn't need to remap the variable in `prepareVariables` up above; it would naturally flow in from the `path: "/fair-organizer/:slug"` up above.